app/chrome/manifest.json

```diff
@@ -23,8 +23,7 @@
 
   "offline_enabled": true,
 
-  "content_security_policy": "script-src 'self' 'unsafe-eval' 'unsafe-inline' https://maps.googleapis.com; object-src 'self' 'unsafe-eval' 'unsafe-inline'",
-
+  "content_security_policy": "script-src 'self' https://www.googletagmanager.com https://maps.googleapis.com https://www.google-analytics.com https://ssl.google-analytics.com; object-src 'self'",

```

Comment:
@dfarache why did you remove the `unsafe-eval` and `unsafe-inline`? or rather did you tested the complete simulator to see if it works like that?

Reply:
I removed unsafe-inline because it seems to be considered unsafe and ignored by Chrome as you may see in the screenshot below.

Then, unsafe-eval does not seem to be needed: our external dependencies (maps and analytics) load correctly without it and it's probably better to disallow it unless it's strictly needed.

![screen shot 2018-08-20 at 13 34 18](https://user-images.githubusercontent.com/5766203/44338293-cbb05b80-a47d-11e8-8418-81d1a9a35533.png)

Reply:
@dfarache the simulator does a lot of client side "eval" so I would prefer if we properly test this before deploying it. 

I have a meeting today with @dkkeller about the test cases for the simulator so it would be good if you could join us